packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -1117,6 +1116,11 @@ abstract class TestRunner {
           version(expected.readTime!)
         );
       }
+
+      const expectedLabels =
```

Comment:
compare actual listen request label with the label generated by expected target purpose.